### ISSUE TITLE
Aria-level setting ignored

### ISSUE THREAD

**Issue by RMurray89** [cgkineo/adapt-tabs#35]
Hi,

I just noticed a minor bug. It looks like the aria-level value set for components in the authoring tool is ignored. I have the tabs component to have an aria-level of 3, but it defaults to level 1.

Thanks
Ross

**Comment by oliverfoster** [comment]
Which version of the framework? Can I have a screenshot of the settings you have and where they are please?

**Comment by RMurray89** [comment]
Hi @oliverfoster,

Sure. It's the latest framework (5.22.1) and the latest AT (latest version).

To be clearer, it's the .tabs__content-item-title-inner heading rather than the component heading.

The code is as follows:

![image](https://user-images.githubusercontent.com/3612643/184089407-b4af8461-2353-4233-99e9-9a901d594378.png)

The settings in the authoring tool (I've actually set it in both the tab component and in configuration settings but makes no difference).

Tab component settings

![image](https://user-images.githubusercontent.com/3612643/184089617-ce9eafd2-9b22-4a3f-a4e1-f28aa0f97412.png)

Configuration settings

![image](https://user-images.githubusercontent.com/3612643/184089843-41efffa5-d8f2-405c-8df4-74b96711cfb7.png)

Component-item aria-level is set at 4. I think I had a reason for this when I was building out course content, but it should be taking the aria-level from the component settings anyway.

Hope this helps,
Ross

**Comment by oliverfoster** [comment]
where did "_aria Level" come from? It's not in the schema.

**Comment by oliverfoster** [comment]
As in, you're in the AAT rewrite, not the classic AAT? 

https://github.com/adaptlearning/adapt-contrib-core/blob/master/schema/content.schema.json#L110

**Comment by RMurray89** [comment]
@oliverfoster - Our Adapt Core is 6.14.2 and authoring tool is 0.10.5...

**Comment by oliverfoster** [comment]
Ahh yes https://github.com/adaptlearning/adapt-contrib-core/blob/4992dfe968988b2239786514eee893b497ae9bac/schema/component.model.schema#L86-L93

**Comment by RMurray89** [comment]
Yeah that looks like the setting not being picked up :)

**Comment by oliverfoster** [comment]
Can I have your course json please? Just bundle it into a zip and upload here. Or private gitter message.

**Comment by RMurray89** [comment]
@oliverfoster - Is the best way to do that to just publish the course then package up the /course folder as a zip yeah?
